src/avrdude.conf.in

```diff
@@ -756,6 +756,19 @@ programmer
     connection_type        = serial;
 ;
 
+#------------------------------------------------------------
+# urclock
+#------------------------------------------------------------
+
+# See https://github.com/stefanrueger/urboot
+programmer
+    id                     = "urclock";
+    desc                   = "Urclock programmer for urboot bootloaders (arduino compatible)";
```

Comment:
Not always compatible with Arduino, right?

Reply:
done!